/**:
  ros__parameters:
    param_b:          false
    params_b:         [true, false, true, true]
    numeric:
      param_i64:      -2                          # [-4, 10], step=2
      param_d:        0.7                         # [-1.0, 2.0]
      enum_param_d:   3.2                         # 1.0 or 2.1 or 3.2
      params_i64:     [-10, 5, 0, 50]
      params_d:       [-4.2, 1.3, 0.2]
    string:
      param_s:        foo
      enum_param_s:   bar
      params_s:       [alpha, beta, gamma]
